### PR TITLE
TicTacToe Sample

### PR DESCRIPTION
**Pull request by edgarfgp**
This PR adds the TictacToe Maui sample initial template . I will adding Widgets and extensions :)

### CODE REVIEW

File: samples/TicTacToe/Platforms/Tizen/Main.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using System;
```
Comment (TimLariviere) [inline samples/TicTacToe/Platforms/Tizen/Main.cs]: Can you replace this file with this one?
https://github.com/fabulousfx/FSharp.Mobile.Templates/blob/main/templates/Maui/Platforms/Tizen/Main.fs

File: samples/TicTacToe/Platforms/Windows/App.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.UI.Xaml;
```
Comment (TimLariviere) [inline samples/TicTacToe/Platforms/Windows/App.xaml.cs]: Can you replace this file with this one?
https://github.com/fabulousfx/FSharp.Mobile.Templates/blob/main/templates/Maui/Platforms/Windows/App.xaml.fs

**Comment by TimLariviere** [comment]
I removed the extra C# files that are not used, but we still have some that can be replaced.

It seems like you did the creation of the project by yourself. Next time, you can directly use `dotnet new maui-fsharp` or duplicate an existing sample and change the names.

File: samples/TicTacToe/Platforms/Tizen/Main.cs (hunk 1 above)
Comment (edgarfgp) [reply samples/TicTacToe/Platforms/Tizen/Main.cs]: Updated

File: samples/TicTacToe/Platforms/Windows/App.xaml.cs (hunk 2 above)
Comment (edgarfgp) [reply samples/TicTacToe/Platforms/Windows/App.xaml.cs]: Done

**Comment by edgarfgp** [comment]
> I removed the extra C# files that are not used, but we still have some that can be replaced.
> 
> It seems like you did the creation of the project by yourself. Next time, you can directly use `dotnet new maui-fsharp` or duplicate an existing sample and change the names.

Ohh I completely forgot about this :( . Thanks for the tip

**Review by TimLariviere — APPROVED** [review]
Great, thanks! 👍

**Pull request merged**